Content.Server/Speech/EntitySystems/BipolarDisorderAccentSystem.cs

```diff
@@ -0,0 +1,69 @@
+using System.Text.RegularExpressions;
+using Content.Server.Speech.Components;
+using Content.Server.Chat.Systems;
+using Robust.Shared.Random;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class BipolarDisorderAccentSystem : EntitySystem
+{
+    [Dependency] private readonly ChatSystem _chat = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    private static readonly Regex RegexDots = new Regex(@"\.{1,3}");
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<BipolarDisorderAccentComponent, AccentGetEvent>(OnAccent);
```

Comment:
nitpick: Usually we add a line between these two parts.

```suggestion
        base.Initialize();

        SubscribeLocalEvent<BipolarDisorderAccentComponent, AccentGetEvent>(OnAccent);
```